receiver/gitmetricsreceiver/config.go

```diff
@@ -0,0 +1,82 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package gitmetricsreceiver // import "github.com/liatrio/otel-liatrio-contrib/receiver/githubreceiver"
+
+import (
+	"errors"
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/gitmetricsreceiver/internal"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/gitmetricsreceiver/internal/metadata"
+)
+
+const (
+	scrapersKey = "scrapers"
+)
+
+// Config that is exposed to this github receiver through the OTEL config.yaml
+type Config struct {
+	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
+	Scrapers                                map[string]internal.Config `mapstructure:"scrapers"`
+	metadata.MetricsBuilderConfig           `mapstructure:",squash"`
+}
+
+var _ component.Config = (*Config)(nil)
+var _ confmap.Unmarshaler = (*Config)(nil)
+
+// Validate the configuration passed through the OTEL config.yaml
+func (cfg *Config) Validate() error {
+	if len(cfg.Scrapers) == 0 {
+		return errors.New("must specify at least one scraper when using git metrics receiver")
```

Comment:
```suggestion
		return errors.New("must specify at least one scraper")
```

I know the `hostmetrics` receiver has similar message, but it's redundant:

```console
Error: invalid configuration: receivers::hostmetrics: must specify at least one scraper when using hostmetrics receiver
```

Reply:
Yea, definitely leveraged the host metrics receiver when converting this to allow for multiple scrapers 😃